rustls-platform-verifier/src/tests/verification_mock/mod.rs

```diff
@@ -94,17 +96,21 @@ const LOCALHOST_IPV6: &str = "::1";
 #[cfg_attr(feature = "ffi-testing", allow(dead_code))]
 pub(super) fn verification_without_mock_root() {
     ensure_global_state();
+    let crypto_provider = CryptoProvider::get_default().unwrap().clone();
```

Comment:
Since we need a line to explicitly grab the crypto provider anyway now (here and the other places in the changeset), I think we can refactor `ensure_global_state` into `get_test_provider`:
```rust
fn get_test_provider() -> rustls::CryptoProvider {
    rustls::crypto::ring::default_provider()
}
```
This also aligns with the other changes in the PR which reduce global/implicit state in the actual library too. Thoughts?

Reply:
Yes, great idea! Done.